### PR TITLE
Change Recovery Services swagger to composite swagger

### PR DESCRIPTION
**Pull request by sarangan12**
Change Recovery Services swagger to composite swagger. 

Ref PR; https://github.com/Azure/azure-rest-api-specs/pull/920 

@dsgouda @vishrutshah Please review and approve.

### CODE REVIEW

**Comment by veronicagg** [comment]
@sarangan12 Changes look good to me. Even thought I see the label "non-breaking-changes" I believe this may introduce breaking changes, as the spec changed, so by pointing to the new versions I expect at least namespaces will be different. Thoughts?
@vishrutshah @sarangan12 I'm curious on Vishrut's comment above, could you summarize the offline conversation on why to not merge this one?

**Comment by dsgouda** [comment]
I'm not sure what implications this would have on the ruby sdk but I am quite certain this is a breaking change for C# sdk

**Comment by dsgouda** [comment]
@sarangan12 Please hold off on this PR for now, we ran into some issues with the original PR and we have had to revert it, will post here once we are good to go

**Comment by vishrutshah** [comment]
@sarangan12 Can you please summarize what we discussed here, please.

**Comment by dsgouda** [comment]
@sarangan12 Please merge these changes to appease the changes here https://github.com/Azure/azure-rest-api-specs/pull/1087
